materials/models/noz/partyhat.vmt

```diff
@@ -1,4 +1,4 @@
-"VertexlitGeneric"
-{
-	"$basetexture" "models/misc/PartyHat.vtf"
+"VertexlitGeneric"
+{
+	"$basetexture" "models/misc/PartyHat.vtf"
```

Comment:
need to rename this

Reply:
I had forgotten to change all vmt data for all the materials